src/Idris/Parser/Helpers.hs

```diff
@@ -276,7 +276,7 @@ idrisStyle = IdentifierStyle {
       ["let", "in", "data", "codata", "record", "corecord", "Type", "do", "dsl",
       "import", "impossible", "case", "of", "total", "partial", "mutual",
       "infix", "infixl", "infixr", "rewrite", "where", "with", "syntax",
-      "proof", "postulate", "using", "namespace", "class", "instance",
+      "proof", "postulate", "using", "namespace", "instance",
```

Comment:
We should not yet. While it is deprecated now and people should change from it, I don't think that this is the right PR to break support.